app/src/main/java/com/pileproject/drive/util/bluetooth/RxBluetoothConnector.java

```diff
@@ -77,11 +81,47 @@ public void call(Subscriber<? super BluetoothSocket> subscriber) {
 
                     subscriber.onNext(socket);
                     subscriber.onCompleted();
-                } catch (IOException e) {
-                    subscriber.onError(e);
+                } catch (IOException firstCause) {
+
+                    try {
+                        BluetoothSocket socket = connectByRfcommSocket(bluetoothDevice);
+                        subscriber.onNext(socket);
```

Comment:
You're right haha.

By the way this method is not used at this point; I didn't intend to include this file 
because I couldn't modify BluetoothCommunicator#open so as to use this method.
Do you think this hunk is needed for this PR?